### ISSUE TITLE
Wrong workspaces setup with commitlint results to `depencies.map is not a function`

### ISSUE THREAD

**Issue by ElPrudi** [webpro/knip#48]
**Knip version**: 

### Bug 

Setting up multiple workspaces in a monorepo the wrong way results in this error message:

```
Resolving plugin configuration files...
<project-path>/knip/node_modules/knip/dist/workspace-worker.js:236
            return dependencies.map(symbol => ({ type: 'unlisted', filePath: configFilePath, symbol }));
                                ^

TypeError: dependencies.map is not a function
    at <project-path>/knip/node_modules/knip/dist/workspace-worker.js:236:33
    at async Promise.all (index 0)
    at async WorkspaceWorker.findDependenciesByPlugin (<project-path>/knip/node_modules/knip/dist/workspace-worker.js:227:45)
    at async WorkspaceWorker.findDependenciesByPlugins (<project-path>/knip/node_modules/knip/dist/workspace-worker.js:205:42)
    at async main (<project-path>/knip/node_modules/knip/dist/index.js:186:56)
    at async run (<project-path>/knip/node_modules/knip/dist/cli.js:21:46)
    at async <project-path>/knip/node_modules/knip/dist/cli.js:48:1
```

### How to reproduce

- Omit `"workspaces": []` in `package.json`
- Omit `"workspaces": {}` in custom `knip.json`
- Set wrong directory names in `package.json` or `knip.json`
- ignore everything, so it can't lint anything
- and maybe more cases

### Workaround

Idk, this monorepo functionality is quite buggy, which is why I can't really use knip :(

**Comment by ElPrudi** [comment]
Ok, i found out the culprit. I have a custom commitlint package and setting it to false in my `knip.json` resolved the problem.

**Comment by webpro** [comment]
This might be fixed in [v1.7.1](https://github.com/webpro/knip/releases/tag/1.7.1)

**Comment by ElPrudi** [comment]
Can confirm. v1.7.1 fixed both `dependencies.map is not a function` error and the problem that it only linted the commtlint package.
